tests/api/v2/handlers/test_planners_api.py

```diff
@@ -42,3 +50,10 @@ async def test_unauthorized_get_planner_by_id(self, api_v2_client, test_planner)
     async def test_get_nonexistent_planner_by_id(self, api_v2_client, api_cookies):
         resp = await api_v2_client.get('/api/v2/planners/999', cookies=api_cookies)
         assert resp.status == HTTPStatus.NOT_FOUND
+
+    async def test_planner_defaults(self, api_v2_client, api_cookies, test_planner, test_planner_2):
```

Comment:
This tests the API with NO defaults set right?

```suggestion
    async def test_planner_no_default(self, api_v2_client, api_cookies, test_planner, test_planner_2):
```

Reply:
No, actually - this is a case where both test planners are provided to the API by the test's config, and since "123test planner" is less than "atomic" when sorted (1 < a in string comparisons), the only way atomic shows up first is if a matching default value ("atomic") is present in the configuration.

Basically, this is a really sneaky way of leveraging the default configuration to validate that the configuration has the intended effect. I can make changes to make this more explicit, if you'd like, but it's definitely a `test_planner_default` case, since the default is present in the base config.

Reply:
Ah okay.

--Out of scope question. Is the return of defaults purely for UI to show pre-highlighted fields?

Reply:
Pretty much.